db.go

```diff
@@ -44,13 +44,113 @@ func init() {
 	db.Exec(`
```

Comment:
@cglotr I really like that our db/repository functions are very minimal. I think that's good especially in update statements.